server/pbenchinacan/run-pbench-in-a-can

```diff
@@ -1,5 +1,25 @@
 #!/bin/bash
 
+#
+# Specification for a self-contained Pbench Server Environment.
+#
+# This configuration creates a pod for the Pbench Server deployment using
+# four separate containers: the Pbench Server and its associated
+# Elasticsearch instance, PostgreSQL database instance, and KeyCloak broker
+# instance. It then runs the server/pbenchinacan/load_keycloak.sh to
+# configure the keycloak broker for the minimum requirements of the
+# Pbench-Server.
+#
+# The pod should be invoked via the following command
+#    server/pbenchinacan/run-pbench-in-a-can
```

Comment:
I don't think that this comment provides enough value to justify retaining it (e.g., if we rename the script, then this comment will be incorrect; on the other hand, if the reader can find this comment, then they already know what the path to the script is).

If you want to say that this script should be run from the root of the Git checkout, then say _that_ instead -- don't just name the script here (because that doesn't necessarily make clear the requirement of the current directory).